app/src/main/java/io/pumpkinz/pumpkinreader/etc/ShareBroadcastReceiver.java

```diff
@@ -0,0 +1,25 @@
+package io.pumpkinz.pumpkinreader.etc;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+
+import org.parceler.Parcels;
+
+import io.pumpkinz.pumpkinreader.model.News;
+import io.pumpkinz.pumpkinreader.util.ActionUtil;
+
+
+public class ShareBroadcastReceiver extends BroadcastReceiver {
+
+    @Override
+    public void onReceive(Context context, Intent intent) {
+        News news = Parcels.unwrap(intent.getParcelableExtra(Constants.NEWS));
+
+        Intent chooser = Intent.createChooser(ActionUtil.getPumpkinShareIntent(context, news), "Share");
```

Comment:
Oh, my bad, you need `FLAG_ACTIVITY_NEW_TASK`. Ok ok.

For `Share` string, how about using string from `strings.xml`?